cw/cw305/kmac_cw_demo.py

```diff
@@ -0,0 +1,65 @@
+# Copyright lowRISC contributors
```

Comment:
I am not sure if this file would work at the moment (I mean the interfacing with the capture and target board etc.) and you probably also can't test this atm. To verify the XKCP interface, you have implemented a separate test function. So I am inclined to not merge this in the first place. What is your view?

Reply:
agree